app/apps/itinerary/models.py

```diff
@@ -348,6 +350,11 @@ def save(self, *args, **kwargs):
 
         super().save(*args, **kwargs)
 
+    def get_visits_today(self, *args, **kwargs):
+        today = timezone.now().replace(hour=0, minute=0, second=0)
+        tomorrow = today + datetime.timedelta(days=1)
```

Comment:
Ik vraag me af of we dit nodig hebben. Een itinerary en itinerary item wordt eigenlijk altijd maar voor één dag gebruikt. Elke ochtend krijgt de gebruiker een nieuw beginscherm te zien waarin een nieuwe lijst wordt gegenereerd. Het zal niet zo zijn dat een gebruiker met een looplijst van vorige week gaat lopen.

Dus wat de betreft is een reverse accessor hier eigenlijk al voldoende (self.visits) en mag visits_today in de serializer ook gewoon visits zijn

Reply:
Thanks voor de verduidelijking! pas het aan